src/main/haskell/kore/src/Kore/Step/Simplification/Ceil.hs

```diff
@@ -190,9 +193,71 @@ makeEvaluateTerm
         (result, SimplificationProof)
   where
     headAttributes = MetadataTools.symAttributes tools patternHead
+makeEvaluateTerm
+    tools
+    (DV_ _ child)
+  =
+    makeEvaluateBuiltinDomain tools child
 makeEvaluateTerm
     tools term
   =
     ( give (MetadataTools.symbolOrAliasSorts tools ) $ makeCeilPredicate term
     , SimplificationProof
     )
+
+{-| Evaluates the ceil of a domain value.
+-}
+makeEvaluateBuiltinDomain
+    :: forall level variable .
+        ( level ~ Object
+        , SortedVariable variable
+        , Eq (variable level)
+        , Show (variable level)
+        )
+    => MetadataTools level StepperAttributes
+    -> BuiltinDomain (PureMLPattern level variable)
+    -> (Predicate level variable, SimplificationProof level)
+makeEvaluateBuiltinDomain
+    _tools
+    (BuiltinDomainPattern (StringLiteral_ _))
```

Comment:
You might add a note that this is true if-and-only-if the pattern has passed the verifier. Of course, that should be true at this point. Eventually, I don't think we should even have string-literal domain values at this point in the stepper.